ska_helpers/version.py

```diff
@@ -97,13 +146,31 @@ def get_version(package, distribution=None):
         import traceback
         import warnings
 
+        version = "0.0.0"
+        log(f"FAIL: got {version=}")
```

Comment:
My only nitpick so far is that this is not necessarily an error but a warning, since the code does not fail. Also, this message is redundant considering the warning below.

My comments below say that, when running testr with (the remaining) namespace packages, you will always see this error message that will say FAIL (and fail the post-check-logs). This is not something we want, which is why the warning is issued only within the conditional block that follows.

Originally, I had the warning right where you added the log statement, and that made all `Ska.*` tests fail. That does not seem to be the case anymore, and perhaps we do not need the conditional statement, but we would still need the warning, since this is something that should not happen in general.

Right now I tried this module and I never saw the `FAIL` message, and perhaps it has to do with other changes we made.

Reply:
> Originally, I had the warning right where you added the log statement, and that made all Ska.* tests fail. 

The logging is not enabled by default, so it will be quiet unless it ends up not finding a valid version and using 0.0.0. So if I understand what you said then everything is consistent.